proto/harpocrate.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package gravitalia.harpocrate.v1;
+
+option csharp_namespace = "Gravitalia.Harpocrate.V1";
+option go_package = "github.com/gravitalia/harpocrate/proto";
+option java_package = "com.gravitalia.harpocrate.v1";
+option java_multiple_files = true;
+option java_outer_classname = "HarpocrateProto";
+
+// The shortener service creation that allows to reduce URL length
+service Harpocrate {
+    // Reduce link length by according a random ID to a link
+    rpc Reduce (ReduceRequest) returns (ReduceReponse) {}
+}
+
+// Enumerate every possible options
+enum Option {
+    // Disable analytics and url analysis
+    DISABLE_BOTH = 0;

```

Comment:
In enumerations, the `0`st variant is the default value used by parsers when a value is not set. This might be problematic within the code.
You should increment all of them by one and add a default value, such as:

```proto
enum Option {
    // Unspecified enum value, implementations should therefore apply no analytics
    // for safety reasons.
    OPTION_UNSPECIFIED = 0;
}
```